plugins/azure_sentinel/help.md

```diff
@@ -511,190 +651,105 @@ _This plugin does not contain any triggers._
 
 ### Custom Output Types
 
-#### Alert
```

Comment:
Can these custom output types be added back in to the help.md if they are still used?